### DISCUSSION TITLE
Identifier name is too long when creating relation between tables

### DISCUSSION THREAD

**Discussion by eugeneihde** [nocodb/nocodb#8691] [Q&A]
I want to create a relation between to tables in nocodb. So i got into the table and created a new field with the field type `Links`. I selected the `Has Many` Relationship and selected the table i wanted to link to. Now when i tried to save i got this error message:

> alter table `nc_v76b___StrassenanschriftInland` add index `nc_v76b___strassenanschriftinland_nc_v76b___codeartanschrift_id10_index`(`nc_v76b___CodeArtAnschrift_id10`) - Identifier name 'nc_v76b___strassenanschriftinland_nc_v76b___codeartanschrift_id10_index' is too long

Is there a way to fix this?

**Comment by o1lab** [comment]
What is the underlying database - can you provide the [project info](https://docs.nocodb.com/faqs/#how-to-check-my-project-info-) ?

**Reply by eugeneihde** [reply]
Node: **v20.12.1**
Arch: **x64**
Platform: **linux**
Docker: **false**
RootDB: **mysql2**
PackageVersion: **0.207.3**

**Reply by o1lab** [reply]
<img width="760" alt="image" src="https://github.com/nocodb/nocodb/assets/5435402/8d3602bc-a423-4c24-b3b9-6c3ec248bca3">

You can [tweak mysql conf](https://laracasts.com/discuss/channels/laravel/indexing-error-on-identifier-name-is-too-long) for now.

**Reply by eugeneihde** [reply]
This didnt work. We have a MariaDB v11.0.2

**Reply by o1lab** [reply]
[Issue has been created for this](https://github.com/nocodb/nocodb/issues/8692)
